pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml

```diff
@@ -37,7 +37,6 @@ spec:
         - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
         - -skip-auth-regex=^/metrics
         image: '*** GENERATED ***'
-        imagePullPolicy: Always
```

Comment:
for my understanding, why did these go away?  Looks like the default is to `IfNotPresent` which causes the kubelet to skip pulling if it exists vs forcing a pull each time.

Src:
https://kubernetes.io/docs/concepts/containers/images/

Reply:
Yes, this isn't right. https://github.com/openshift/openshift-azure/issues/120.